mobile/lib/screens/dashboard/dashboard_view.dart

```diff
@@ -341,45 +335,15 @@ class _DashboardViewState extends State<DashboardView>
                     ),
                     BlocBuilder<KyaBloc, KyaState>(
                       builder: (context, state) {
-                        List<Quiz> inCompleteQuizzes = state.quizzes
-                            .where((quiz) => quiz.status != QuizStatus.complete)
-                            .toList();
+                        List<Quiz> quizzes = state.quizzes;
```

Comment:
@katendejericho5 I don't think the filtering should be removed. It can however be adjusted to prioritize incomplete quizzes first